### PR TITLE
Removecolumn

### LINKED ISSUE #10418: Remove `name` column from `categories` table
Follow-up for https://github.com/mozilla/addons-server/issues/10392 : once that is fixed in prod, we can remove the `name` column from the `category` table entirely.

### PR DESCRIPTION
**Pull request by soniasingla**
Fixes #10418

### CODE REVIEW

File: src/olympia/migrations/1068-remove-categories-table-users.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE 'categories';
+DROP COLUMN 'name';
```
Comment (eviljeff) [inline src/olympia/migrations/1068-remove-categories-table-users.sql]: In sql column and table names should be quoted with `` ` `` not ` ' `. 
Also, you need an empty line at the end of the file

File: src/olympia/devhub/templates/devhub/new-landing/components/manage_your_addon.html [hunk 2]
```diff
@@ -6,18 +6,17 @@ <h3 class="DevHub-content-header DevHub-content-header--manage">{{ _('Manage You
...
+          a few small changes. Learn more about the differences in 
+          extension development between the four major browsers and 
+          how you can make your extension cross-browser compatible.
           {% endtrans %}
         </p>
-        <a href="https://developer.mozilla.org/en-US/Add-ons/Distribution/Resources_for_publishers" class="Button Button--evolution">{{ _('View Resources for Publishers') }}</a>
+        <a href="https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Build_a_cross_browser_extension" class="Button Button--evolution">{{ _('View Best Practices for Cross-Browser Development') }}</a>
       </div>
     </div>
   </div>
```
Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/new-landing/components/manage_your_addon.html]: What's all this. It seems unrelated to the issue.

**Comment by diox** [comment]
(In any case this should not be merged until after we've tagged 2018.01.24)

**Comment by EnTeQuAk** [comment]
@soniasingla do you have time to clean up this pull request and work on the suggested changes?

**Comment by EnTeQuAk** [comment]
You don't have to close this pull request. Just do your changes in the branch you already used and push your changes. The pull request will reflect those updates.

That will make our lives much easier because creating new pull requests destroys existing review context and makes it harder to see what we discussed earlier etc.

File: src/olympia/migrations/1068.remove-name-column.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `categories`
+ DROP COLUMN `name`;
```
Comment (championshuttler) [inline src/olympia/migrations/1068.remove-name-column.sql]: you need a blank line after this @soniasingla 😉

Comment (championshuttler) [inline src/olympia/migrations/1068.remove-name-column.sql]:
```suggestion
 DROP COLUMN `name`;
 
```

**Comment by soniasingla** [comment]
Thanks for the help its ready for another review :smile:

**Comment by eviljeff** [comment]
the filename needs the number increasing to 1070 from 1068 (1068 and 1069 were taken by other patches in the meantime).  It should be `1070-remove-name-column.sql`
